### PR TITLE
Census Endpoint

### PR DESCRIPTION
**Pull request by ZadenRB**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,5 +22,6 @@ vendor
 
 .env
 api/api
+census_acs5/census_acs5
```
Comment (dviramontes) [inline .gitignore:25]: is this a file we are gitignoring ?

File: Makefile [hunk 2]
```diff
@@ -25,12 +27,24 @@ dev:
 .PHONY: test
 test:
 	go test -v ./api
-	go test -v ./geography
+	go test -v ./census_acs5
+  go test -v ./goegraphy
```
Comment (dviramontes) [inline Makefile]: indentation is important for makefiles

**Review by dviramontes — CHANGES_REQUESTED** [review]
Looking great, left some comments

File: .gitignore (hunk 1 above)
Comment (ZadenRB) [reply .gitignore:25]: Yes, it’s the binary for this endpoint.

File: Makefile (hunk 2 above)
Comment (ZadenRB) [reply Makefile]: Yeah I’ll fix that when I get the chance, got messed up when editing in github merge conflict editor.

File: Makefile [hunk 3]
```diff
@@ -7,11 +7,13 @@ deps:
 .PHONY: clean
 clean:
 	rm -rf ./api/api
-	rm -rf ./geography/geography
+	rm -rf ./census_acs5/census_acs5
+  rm -rf ./geography/geography
```
Comment (dviramontes) [inline Makefile]: looks like we missed this indent

**Review by dviramontes — APPROVED** [review]
Just the one comment, but otherwise lGTM!

File: Makefile (hunk 3 above)
Comment (dviramontes) [reply Makefile]:
```suggestions
	rm -rf ./census_acs5/census_acs5
	rm -rf ./geography/geography
```

Comment (dviramontes) [inline Makefile]:
```suggestions
	rm -rf ./census_acs5/census_acs5
	rm -rf ./geography/geography
```

**Review by dviramontes — APPROVED** [review]

**Pull request merged**